test/getsettest.c

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
`getsettest`? :) How about `tlsext_status_get_set_test`?

Also can I pretty please ask everyone to write new tests in the unit test framework?